terraform/modules/stack/base/outputs.tf

```diff
@@ -65,11 +65,20 @@ output "rds_postgres_security_group" {
 
 /* RDS Bosh Instance */
 output "bosh_rds_url" {
```

Comment:
So if we name the new instance `rds_curr`, at some point in the future we'll probably want to move it to `rds_prev` and put something else at `rds_curr`. Would terraform be smart enough to understand that rename, or would it delete and recreate the instance?